src/drivers/amd/common/acp_dma.c

```diff
@@ -22,10 +22,15 @@
 #include <stddef.h>
```

Comment:
Can you please add more details about what 'probe' functionality is?

Is the ability to get some statistics/logs  via compress interface?

Reply:
Hi,
This is a debug feature to probe the data in between components of pipeline, for more information refer below 
 https://thesofproject.github.io/latest/developer_guides/debugability/probes/index.html

Reply:
@dbaluta its already used on Intel platforms. Kernel parts also upstream.